xml/src/test/resources/configs/default-serializer.xml

```diff
@@ -14,6 +14,8 @@
   ~ limitations under the License.
   -->
 
+<!-- Please do not format the file, this is there to test #461   -->
```

Comment:
Can we have a test that parses this file in some way that doesn't collapse the whitespace and asserts that the whitespace we expect to see still exists?  (doesn't even have to be an XML parser)